File: app/src/main/java/com/edanichev/nounIcons/app/main/NounHintCloud/Model/HintCloudInteractor.java
package com.edanichev.nounIcons.app.main.NounHintCloud.Model;

public class HintCloudInteractor implements IHintCloudInteractor {

    @Override
    public CloudTagsModel getTags() {

        CloudTagsModel tags = new CloudTagsModel();
        tags.add("cat");
        tags.add("info");
        tags.add("dog");
        return tags;
    }
}

File: app/src/main/java/com/edanichev/nounIcons/app/main/NounHintCloud/Model/HintCloudInteractor.kt
package com.edanichev.nounIcons.app.main.NounHintCloud.Model

open class HintCloudInteractor {

    open fun getTags(): CloudTagsModel {

        val tags = CloudTagsModel()
        tags.add("cat")
        tags.add("info")
        tags.add("dog")
        return tags
    }
}
